public/app/features/explore/Logs/LogsContainer.tsx

```diff
@@ -164,20 +164,28 @@ class LogsContainer extends PureComponent<LogsContainerProps, LogsContainerState
     row: LogRowModel,
     origRow: LogRowModel,
     options: LogRowContextOptions
-  ): Promise<DataQueryResponse | []> => {
+  ): Promise<DataQueryResponse> => {
     const { logsQueries } = this.props;
 
     if (!origRow.dataFrame.refId || !this.state.dsInstances[origRow.dataFrame.refId]) {
-      return Promise.resolve([]);
+      return Promise.resolve({
+        data: [],
+      });
```

Comment:
particularly keen to get a review on this since it changes actual return values instead of just types. @grafana/observability-logs wdyt? 🤔

Reply:
sorry, just realised this should be @grafana/observability-traces-and-profiling

Reply:
This file is owned by @grafana/observability-logs 👍

Reply:
Thanks for pinging! This looks good to me and seems like an error in the original implementation. Pinging @svennergr as he probably implemented this, but seeing how this is used, looks good to me.

Reply:
Yep, LGTM. Just tested and this looks better now! Thanks @ashharrison90 !

Reply:
thanks all 🙇